templates/content.html

```diff
@@ -3,6 +3,6 @@
 
 {% block content %}
     {% placeholder content or %}
-        <p>Hello world! This is HTML5 Boilerplate.</p>
+        <p>This page has no content yet, switch to structure mode and start adding content.</p>
```

Comment:
`<p>This page has no content yet. Switch to <em>Structure mode</em> and start adding content.</p>`

Could we even add an image of the button to help users? Otherwise looks fine.